src/main/java/codeu/controller/LogoutServlet.java

```diff
@@ -0,0 +1,74 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.mindrot.jbcrypt.*;
+
+/** Servlet class responsible for the logout page. */
+public class LogoutServlet extends HttpServlet {
+
+  /** Store class that gives access to Users. */
+  private UserStore userStore;
+
+  /**
+   * Set up state for handling logout-related requests. This method is only called when running in a
+   * server, not when running in a test.
+   */
+  @Override
+  public void init() throws ServletException {
+    super.init();
+    setUserStore(UserStore.getInstance());
+  }
+
+  /**
+   * Sets the UserStore used by this servlet. This function provides a common setup method for use
+   * by the test framework or the servlet's init() function.
+   */
+  void setUserStore(UserStore userStore) {
+    this.userStore = userStore;
+  }
+
+  /**
+   * This function fires when a user requests the /logout URL. It simply forwards the request to
+   * logout.jsp.
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException, ServletException {
+        request.getSession().setAttribute("user", null);
+        response.sendRedirect("/login");
+  }
+
+  /**
+   * This function fires when a user submits the logout form. It gets the username and password
+   * from the submitted form data, checks that they're valid, and either adds the user to the session
+   * so we know the user is logged in or shows an error to the user.
+   */
```

Comment:
You might want to revise this comment so that it doesn't say that 'It gets the username and password
+   * from the submitted form data, checks that they're valid, and either adds the user to the session
+   * so we know the user is logged in'

Reply:
That was careless of me. I basically copied pasted but forgot to fix the comments as well.